CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to AdGuard filters
+
+Short introduction, explain what the ways to contribute are.
+
+## How to work with this repo
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/download) with the ["Adblock" syntax plugin](https://marketplace.visualstudio.com/items?itemName=adguard.adblock).
+2. Clone (or fork and clone) [this repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository).
```

Comment:
Maybe it would be better to add a link to `Clone` word instead of `this repo`, because it may be a bit misleading, I think.

Reply:
Thank you, changed it )